### PR TITLE
Issue #7669: Update docs for MultiVariableDeclaration

### LINKED ISSUE #7669: Update doc for MultipleVariableDeclarations
https://checkstyle.org/config_coding.html#MultipleVariableDeclarations

Add example for each existing config to show what code is ok and what code cause violation.
Good example is https://checkstyle.org/config_naming.html#AbstractClassName
Look at referenced commits of #5832, to see what and how should be changed in code.

Please play with module/check by means of https://checkstyle.org/cmdline.html to understand how it works.

------------

PLEASE READ https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-update-examples

### PR DESCRIPTION
**Pull request by AkMo3**
Fixes #7669

Checkstyle

```
PS H:\CheckstyleCheck> cat .\config.xml
<!DOCTYPE module PUBLIC
  "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
  "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
  <module name="TreeWalker">
    <module name="MultipleVariableDeclarations">
    </module>
  </module>
</module>
PS H:\CheckstyleCheck> cat .\Test.java
public class Test {
  public void myTest() {
    int mid;
    int high;
    // ...

    int lower, higher; // violation
    // ...

    int value,
        index;  //Violation
    // ...

    int place = mid, number = high; // violation, each to be decalared at a new line
  }
}
PS H:\CheckstyleCheck> java -jar H:\GitHub\checkstyle\target\checkstyle-8.38-SNAPSHOT-all.jar -c .\config.xml .\Test.java
Starting audit...
[ERROR] H:\CheckstyleCheck\.\Test.java:7:5: Each variable declaration must be in its own statement. [MultipleVariableDeclarations]
[ERROR] H:\CheckstyleCheck\.\Test.java:10:5: Each variable declaration must be in its own statement. [MultipleVariableDeclarations]
[ERROR] H:\CheckstyleCheck\.\Test.java:14:5: Each variable declaration must be in its own statement. [MultipleVariableDeclarations]
Audit done.
Checkstyle ends with 3 errors.
```

### CODE REVIEW

**Comment by romani** [comment]
@AkMo3 , please do not close PRs, you can push with force to your branch to fix all problems.

**Comment by AkMo3** [comment]
GitHub, Generate website

**Comment by AkMo3** [comment]
@romani force pushing and squashing commits are not able to pass commit message check.

**Comment by AkMo3** [comment]
@romani Sorry for that. I realised that making new PR had actually impacted Travis CI negetively. Is there anything I can do to stop running integration-tests on those?

**Comment by AkMo3** [comment]
Github, generate web site

**Comment by romani** [comment]
>Is there anything I can do to stop running integration-tests on those?

Nothing will stop CI to run all. Whole idea of CI to run all.

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MultipleVariableDeclarationsCheck.java [hunk 1]
```diff
@@ -44,6 +44,22 @@
...
+ * public void myTest() {
+ *     int mid;
+ *     int high;
+ *     // ...
+ *     int lower, higher; // violation
+ *     // ...
+ *     int value,
+ *         index;  //Violation
+ *     // ...
+ *     int place = ar[6], number = ar[place]; // violation, each to be decalared at a new line
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MultipleVariableDeclarationsCheck.java:59]: `int place = mid, ....`

Please make it simple and compilable

**Review by romani — CHANGES_REQUESTED** [review]
Items:

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MultipleVariableDeclarationsCheck.java (hunk 1 above)
Comment (AkMo3) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MultipleVariableDeclarationsCheck.java]: Changes are done, have a look.

Comment (AkMo3) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MultipleVariableDeclarationsCheck.java]: Please, have a look.

**Comment by AkMo3** [comment]
@romani Changes are done. Have a look.

**Comment by AkMo3** [comment]
Github, generate web site.

**Comment by AkMo3** [comment]
Due to some push error, the commits got removed, I will reopen it.

**Comment by AkMo3** [comment]
Well it seems this branch is no longer mergeable. I will have to do a new PR. Sorry for inconvinence.

**Pull request closed without merging**